explainaboard/analysis/analyses_test.py

```diff
@@ -161,9 +161,9 @@ def test_generate_report(self) -> None:
             level="bar",
             features=("feat1", "feat2"),
             combo_counts=[
-                (("aaa", "bbb"), 123),
-                (("iii", "jjj"), 456),
-                (("xxx", "yyy"), 789),
+                (("aaa", "bbb"), 123, list(range(123))),
+                (("iii", "jjj"), 456, list(range(123, 123 + 456))),
+                (("xxx", "yyy"), 789, list(range(123 + 456, 123 + 456 + 789))),
```

Comment:
It looks the test case generates very long lists that is not necessary to check the behavior.
```suggestion
                (("aaa", "bbb"), 3, list(range(3))),
                (("iii", "jjj"), 6, list(range(6))),
                (("xxx", "yyy"), 9, list(range(9))),
```